src/Maestro/ReleasePipelineRunner/ReleasePipelineRunnerItem.cs

```diff
@@ -14,5 +14,8 @@ public sealed class ReleasePipelineRunnerItem
 
         [DataMember]
         public int ChannelId { get; set; }
+
+        [DataMember]
+        public int NumberOfRetriesMade { get; set; }
```

Comment:
[nit][personal preference] just attempts, or retries seems enough

Reply:
I prefer to be more explicit. :-)